.vscode/settings.json

```diff
@@ -1,4 +1,13 @@
 {
     "prettier.semi": false,
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
```

Comment:
This commit just tells VS code to ignore node_modules and friends + makes Jest tests run in serial (b/c they share the same instance of DynamoDB local).